src/passes/RemoveUnusedBrs.cpp

```diff
@@ -1020,31 +1021,35 @@ struct RemoveUnusedBrs : public WalkerPass<PostWalker<RemoveUnusedBrs>> {
       }
     } while (anotherCycle);
 
-    // thread trivial jumps
-    struct JumpThreader : public ControlFlowWalker<JumpThreader> {
-      // map of all value-less breaks and switches going to a block (and not a
-      // loop)
-      std::map<Block*, std::vector<Expression*>> branchesToBlock;
+    // Thread trivial jumps.
+    struct JumpThreader
+      : public PostWalker<JumpThreader,
+                          UnifiedExpressionVisitor<JumpThreader>> {
+      // Map of all value-less breaks and switches going to a block. (For
+      // simplicitly, we store all break targets here, both blocks and loops,
+      // since blocks are 99% of the set of branch targets. Any loops added here
+      // are ignored later.)
+      std::unordered_map<Name, std::vector<Expression*>> branchesToBlock;
```

Comment:
Maybe this is better named `blockToBranches` or `labelToBranches`, given the map order?

Reply:
Good idea, thanks, done.